test/threads_exec.jl

```diff
@@ -1233,6 +1233,7 @@ end
             @testset "throw=true" begin
                 tasks, event = create_tasks()
                 push!(tasks, Threads.@spawn error("Error"))
+                wait(tasks[end]; throw=false)
```

Comment:
Unrelated?

Reply:
This was a race condition in the test that started failing after these changes, for some reason